tutorials/tour/_posts/2017-02-13-mixin-class-composition.md

```diff
@@ -9,28 +9,41 @@ categories: tour
 num: 6
 next-page: anonymous-function-syntax
 previous-page: traits
+prerequisite-knowledge: inheritance, traits, abstract-classes, unified-types
 ---
+Mixin class composition means that a class can reuse the members of one or more mixins (i.e. traits and types).
```

Comment:
I think it would be better to define what we mean by 'mixins' instead of 'mixin class composition'. Also, as far I undertand, you can only mixin 'Traits' so "(i.e. traits and types)" does not make sense to me.